src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { act, render, screen } from '@testing-library/react';
+import { App } from '../App';
+import { MemoryRouter } from 'react-router-dom';
+import ResizeObserver from 'resize-observer-polyfill';
+import { useResizeDetector } from 'react-resize-detector';
+
+jest.mock('../data/api');
+jest.mock('../data/api-lapis');
```

Comment:
This test needs the api mocks from the mutation overview branch, so I rebased it on top of it. We'll have to merge this after the base branch is merged.